prow/crier/reporters/gerrit/reporter.go

```diff
@@ -206,9 +217,16 @@ func (c *Client) Report(pj *v1.ProwJob) ([]*v1.ProwJob, error) {
 	}
 	var reviewLabels map[string]string
 	if reportLabel != "" {
-		vote := client.LBTM
-		if report.success == report.total {
-			vote = client.LGTM
```

Comment:
(did you just redefined them? 😏 )

Reply:
They were only used by this package, so I moved them into here, yes.

Reply:
👍 (didn't check the other file)